src/Components/pokemon/Pokemon.js

```diff
@@ -0,0 +1,48 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import "./Pokemon.css";
+
+const Pokemon =({endpoint}) => {
+
+    const [pokemonData, setPokemonData] = useState({});
+
+    useEffect(()=> {
+
+        async function fetchData() {
+
+            try {
+                const response = await axios.get(endpoint);
+                setPokemonData(response.data);
+                console.log(response.data);
+            }catch (e) {
+                console.error(e);
+            }
+        }
+        if(endpoint){
+            fetchData();
+        }
+
+    },[endpoint])
+
+    return (
+        <div className='PokemonDetails'>
+            {Object.keys(pokemonData).length > 0 &&
```

Comment:
Deze kan dan dus simpeler
```suggestion
            {pokemonData &&
```